ForumSessionProvider/extension.json

```diff
@@ -30,8 +30,8 @@
   },
   "AuthManagerAutoConfig": {
   	"primaryauth": {
-		"AuthManager_SMF": {
-		  "class": "AuthManager_SMF",
+		"AuthManager_ForumSoftware": {

```

Comment:
Are you sure there are no hooks to add for this here?

Reply:
MW code is difficult to read but I didn't see any when poking around.  The best way I could see was to create the class that uses the standard auth as its base.